dart/lib/src/sentry.dart

```diff
@@ -132,7 +124,7 @@ class Sentry {
 
   /// Reports an [event] to Sentry.io.
   static Future<SentryId> captureEvent(
-    SentryEvent event, {
+    SentryEvent? event, {
```

Comment:
I guess Ive made the comments for this class already in the Hub class, which is pretty much the same